### PR TITLE
added Sleepy Cow Herding hints

### PR DESCRIPTION
**Pull request by yiminghuang47**
_If any of the below don't apply to this Pull Request, mark the checkbox as done._

- [ ] I have tested my code.
- [ ] I have added my solution according to the steps [here](https://usaco.guide/general/adding-solution#steps).
- [ ] I have followed the code conventions mentioned [here](https://usaco.guide/general/adding-solution/#code-conventions).
  - I understand that if it is clear that I have not attempted to follow these conventions, my PR will be closed.
  - If changes are requested, I will re-request a review after addressing them.

### CODE REVIEW

File: solutions/bronze/usaco-915.mdx [hunk 1]
```diff
@@ -2,11 +2,41 @@
...
 title: Sleepy Cow Herding
-author: Sathvik Chundru, Ryan Chou, Vikas Thoutam
+author: Sathvik Chundru, Ryan Chou, Vikas Thoutam， Yiming Huang
 ---
 
 [Official Analysis (C++)](http://www.usaco.org/current/data/sol_herding_bronze_feb19.html)
 
+<Spoiler title="Hint 1 (Minimum)">
+
+There are only a few possible answers. How many moves does it take at most to group the cows in consecutive locations?
```
Comment (SansPapyrus683) [inline solutions/bronze/usaco-915.mdx:12]: > few possible answers

for what?

File: solutions/bronze/usaco-915.mdx [hunk 2]
```diff
@@ -2,11 +2,41 @@
...
+
+</Spoiler>
+
+<Spoiler title="Hint 2 (Minimum)">
+
+It takes at most 2 moves.
```
Comment (SansPapyrus683) [inline solutions/bronze/usaco-915.mdx:18]: hints shouldn't just give the answer

File: solutions/bronze/usaco-915.mdx [hunk 3]
```diff
@@ -2,11 +2,41 @@
...
+<Spoiler title="Hint 2 (Maximum)">
+
+When there are two consecutive cows, what is the optimal way to move the cow that minimizes the amount of gaps reduced?
+
+</Spoiler>
+
+<Spoiler title="Hint 3 (Maximum)">
+
+Let's say the cows are at location a $<$ b $<$ c where b and c are next to each other, we can move c one position to the left of b. This will only reduce the gap by 1.
+</Spoiler>
```
Comment (SansPapyrus683) [inline solutions/bronze/usaco-915.mdx:37]: same here

**Review by SansPapyrus683 — CHANGES_REQUESTED** [review]

**Comment by yiminghuang47** [comment]
I'm not really sure how much info I should reveal in the hints. I might've deleted too much lol

File: solutions/bronze/usaco-915.mdx (hunk 2 above)
Comment (yiminghuang47) [reply solutions/bronze/usaco-915.mdx]: I see

**Review by SansPapyrus683 — APPROVED** [review]

File: solutions/bronze/usaco-915.mdx [hunk 4]
```diff
@@ -2,31 +2,9 @@
 id: usaco-915
 source: USACO Bronze 2019 February
 title: Sleepy Cow Herding
-author: Sathvik Chundru, Ryan Chou, Vikas Thoutam， Yiming Huang
+author: Sathvik Chundru, Ryan Chou, Vikas Thoutam Yiming Huang
```
Comment (yiminghuang47) [inline solutions/bronze/usaco-915.mdx:5]: wait why did you delete the comma lol

Comment (SansPapyrus683) [reply solutions/bronze/usaco-915.mdx:5]: i thought i deleted the space what

**Review by ChuangSheep — APPROVED** [review]
looks good

**Pull request merged**